.github/NativeBuildReport.kts

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env kscript
+
+@file:MavenRepository("jcenter","https://jcenter.bintray.com/")
+@file:MavenRepository("maven-central","https://repo1.maven.org/maven2/")
+@file:DependsOn("org.kohsuke:github-api:1.101")
+
+
+import org.kohsuke.github.*
+import java.io.File
+import java.io.IOException
+import java.util.concurrent.TimeUnit
+
+val token = args[0]; 
+val status = args[1];
+
+
+// Handle status. Possible values are success, failure, or cancelled.
+val succeed = status == "success";
+if (status == "cancelled") {
+    println("Job status is `cancelled` - exiting")
+    System.exit(0)
+}
+
+val ISSUE_TITLE = "Quickstarts Native Build failed against Quarkus Master"
+val REPO = "quarkusio/quarkus-quickstarts"
+
+val github = GitHubBuilder().withOAuthToken(token).build()
+val repository = github.getRepository("quarkusio/quarkus-quickstarts")
```

Comment:
I think we should open it on quarkusio/quarkus because I am afraid not enough people have eyeballs on the quarkus-quickstart project issues

Reply:
Yes, it should work. I need to check that the token I used has access to this repo.

Reply:
Should work with the current token.